js/modules/k6/http/http.go

```diff
@@ -181,3 +187,12 @@ type Client struct {
 	moduleInstance   *ModuleInstance
 	responseCallback func(int) bool
 }
+
+func checkQuantityParameter(args ...goja.Value) bool {
+	for i := range args {
+		if i >= 1 {
+			return true
+		}
+	}
+	return false
```

Comment:
Não me parece adequado, @juliaokmenezes . O que você está checando aqui? Se o tamanho de args é maior ou igual a 1? Por que não simplesmente `return len(args) >= 1`? Se for isso, acho que nem do método precisa.

Reply:
Eu procurei fazer do jeito que o senhor indicou, contudo o tipo goja não aceita a função len(), eu teria que fazer algumas adaptações para poder funcionar. Contudo, em relação a criação de um método, foi um pedido dos próprios mantenedores do repositório, o que seria interessante, seria eu tirar a verificação do método get e head, e transferir para essa função.